src/types/redis_hash.cc

```diff
@@ -250,28 +254,31 @@ rocksdb::Status Hash::MSet(const Slice &user_key, const std::vector<FieldValue>
   auto batch = storage_->GetWriteBatchBase();
   WriteBatchLogData log_data(kRedisHash);
   batch->PutLogData(log_data.Encode());
+  std::unordered_set<std::string> field_set;
+  for (auto it = field_values.rbegin(); it != field_values.rend(); it++) {
```

Comment:
Any reason to iterator the set in reverse order?

Reply:
If duplicates appear,  we have to use the last one of same member. Like in `hset hkey f1 v1 f1 v11`, 'f1 v11' should be kept.

Reply:
Thanks. Got it.